osu.Server.DifficultyCalculator/Database.cs

```diff
@@ -9,20 +10,25 @@ public class Database
     {
         public static MySqlConnection GetConnection()
         {
-            if (AppSettings.ConnectionStringMaster == null)
-                return null;
+            string host = (Environment.GetEnvironmentVariable("DB_HOST") ?? "db");
```

Comment:
Is there a reason why these are in here and not AppSettings?

Reply:
copying code from queue-processor. i can move i guess?

Reply:
Mmm I guess it's fine if it's documented either way.